cmd/serverless/main.go

```diff
@@ -85,7 +84,10 @@ where they can be graphed on dashboards. The Datadog Serverless Agent implements
 
 	logsLogsTypeSubscribed = "DD_LOGS_CONFIG_LAMBDA_LOGS_TYPE"
 
-	datadogConfigPath        = "datadog.yaml"
+	// AWS Lambda is writing the Lambda function files in /var/task, we want the
```

Comment:
Is this definitive from AWS ? Could we let user modify this through an env var (without documenting it for now). This could be useful if one day AWS chose to change this to avoid a force upgrade of the agent.

Reply:
I think it's fairly definitive indeed since it is how it works and how it is documented on AWS Lambda environments, it was already the case even before the appearance of the extension stuff. We would probably be noticed in time if such a big change would occur (+ releasing a new layer for that is shorter than a full agent release for now).